tenseal/tensors/ckksvector.cpp

```diff
@@ -412,4 +415,60 @@ CKKSVector& CKKSVector::replicate_first_slot_inplace(size_t n) {
     return *this;
 }
 
+CKKSVector CKKSVector::polyval(const vector<double>& coefficients) {
+    CKKSVector new_vector = *this;
+    return new_vector.polyval_inplace(coefficients);
+}
+
+CKKSVector& CKKSVector::polyval_inplace(const vector<double>& coefficients) {
+    if (coefficients.size() == 0) {
+        throw invalid_argument(
+            "the coefficients vector need to have at least one element");
+    }
+
+    int degree = coefficients.size() - 1;
```

Comment:
How should we do that?

Reply:
My confusion here is that coefficients.size() returns size_t, which is an unsigned int64.

Assigning to int should raise some warnings from any compiler, I think.

And you can use 
```
int degree = static_cast<int>(coefficients.size()) - 1;
```

to prevent the warnings.

Reply:
Ah yeah of course, didn't saw that I didn't cast, thank you!